src/mame/namco/namcops2.cpp

```diff
@@ -614,11 +617,17 @@ V290 FCB PCB
 
 V290 FCB PCB is almost identical to FCA PCB. The main differences are changed internal MCU code & PIC code,
 some extra/different connectors, less D1017 driver transistors and an added RS-232 IC.
-The V290 FCB PCB is used with touchscreen games such as Dragon Chronicles, Druaga Online, Idol Master etc.
+The V290 FCB PCB is used with touchscreen games such as Dragon Chronicles, Druaga Online, THE IDOLM@STER etc.
 It supports a serial touchscreen interface, card readers and buttons.
 The additional devices are supported via J108 which connects to another PCB 'XOU020-A' which contains a
 Texas Instruments TMS32VC540x DSP, TSOP32 flash ROM and other components.
 
+V300 FCB PCB
+2606960100 (2582960101)
+
+Notes: V300 FCB PCB is almost identical to the V290 FCB PCB. The main differences are changed internal MCU 
+code & PIC code, more output transistors and an traded the RS-232 for a RS-232C IC.
+[to-do]
 
```

Comment:
"and an traded the RS-232 for a RS-232C IC". What? Come on at least write proper English/grammar. Anyway this part quoted is not required. In practical terms for emulation purposes there is no difference between RS-232 and RS-232C

Reply:
Thanks for the _destructive_ commentary. English is not my first language. 
I copied a note that was similar and lots of people look at MAME's documentation to find quirks or details about some hardware, so I included it for completion's sake. No need to be like that.
Will edit.

Reply:
There are other emulators that utilize MAME drivers and documentation as a basis for naming conventions and guidelines regarding ROM/dump names. I've used them myself as a reference to identify the necessary components for running my own 2x6 hardware. I don't believe it's 'pointless', and I hope other reviewers share the same perspective.

Reply:
> Anyway your whole addition is pointless as PS2 emulation will never be working in MAME to any level of playability 

Nobody in the dev circle really cared so far, already replied to this in the past, it's not a `never` ...

> In future if you are going to make changes to existing docs

I'm honestly of the opinion that these super verbose docs can live as separate markdown files in 2023. They don't help at all when you have to jump back and forth between multi thousand lines of headers.